### ISSUE TITLE
Issues when running locally from node_modules.

### ISSUE THREAD

**Issue by jonniespratley** [PolymerLabs/polyserve#66]
```
/node_modules/.bin/polyserve
module.js:341
    throw err;
    ^

Error: Cannot find module '/Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/polyserve/lib/polyserve.js/lib/cli'
    at Function.Module._resolveFilename (module.js:339:15)
    at Function.Module._load (module.js:290:25)
    at Module.require (module.js:367:17)
    at require (internal/module.js:16:19)
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/polyserve/bin/polyserve:20:13
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:44:21
    at ondir (/Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:187:31)
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:153:39
    at onex (/Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:93:22)
    at /Users/predixgo/GithubEnterpriseOrgs/PredixComponents/px-table-view/node_modules/resolve/lib/async.js:24:18
```

Also whats up with the issues? PolymerLabs needs to fix these issues, this is pretty ridiculous.

**Comment by justinfagnani** [comment]
I'm looking into this. Thanks for your patience and politeness.

**Comment by convolu** [comment]
Hello, I have the same issue on a fresh install from a few minutes ago.

**Comment by justinfagnani** [comment]
I have a fix in #68 Just waiting for a review to merge and publish. In the mean time you can use 0.7.x

**Comment by justinfagnani** [comment]
Fixed in #68

**Comment by convolu** [comment]
Thanks for the prompt response @justinfagnani.
